index.bs

```diff
@@ -57,15 +57,62 @@ spec: mimesniff; urlPrefix: https://mimesniff.spec.whatwg.org/#
     type: dfn; text: valid mime type; url: valid-mime-type
 
 spec: webidl; urlPrefix: https://heycam.github.io/webidl/#
-    type: dfn; text: present
+    type: dfn; text: present; url:dfn-present
+
+spec: dom; urlPrefix: https://www.w3.org/TR/dom/#
+    type: dfn; text: Document; url:concept-document
+
+spec: html52; urlPrefix: https://www.w3.org/TR/html52/
+    type: dfn; text: origin; url:browsers.html#concept-cross-origin
+
+
+spec: encrypted-media; urlPrefix: https://www.w3.org/TR/encrypted-media/#
+    type: dfn
+        for: EME; text: encrypted media
+    type: dfn
+        for: EME; text: Key System; url: key-system
+    type: interface
+        for: EME; text: MediaKeySystemAccess; url: mediakeysystemaccess-interface
+    type: interface
+        for: EME; text: MediaKeys; url: mediakeys-interface
+    type: attribute
+        for: EME; text: keySystem; url: dom-mediakeysystemaccess-keysystem
+    type: attribute
+        for: EME; text: initDataTypes; url: dom-mediakeysystemconfiguration-initdatatypes
+    type: attribute
+        for: EME; text: robustness; url: dom-mediakeysystemmediacapability-robustness
+    type: attribute
+        for: EME; text: distinctiveIdentifier; url: dom-mediakeysystemconfiguration-distinctiveidentifier
+    type: attribute
+        for: EME; text: persistentState; url: dom-mediakeysystemconfiguration-persistentstate
+    type: attribute
+        for: EME; text: sessionTypes; url: dom-mediakeysystemconfiguration-sessiontypes
+    type: interface
+        for: EME; text: MediaKeySystemConfiguration; url: mediakeysystemconfiguration-dictionary
+    type: interface
+        for: EME; text: requestMediaKeySystemAccess(); url: navigator-extension:-requestmediakeysystemaccess()
+    type: dfn
+        for: EME; text: Get Supported Configuration; url: get-supported-configuration
+    type: interface
+        for: EME; text: MediaKeySystemMediaCapability; url: mediakeysystemmediacapability-dictionary
+
+spec: secure-contexts; urlPrefix: https://www.w3.org/TR/secure-contexts/
+    type: dfn; text: Is the environment settings object settings a secure context?; url: #settings-object 
 </pre>
+
 <pre class='biblio'>
 {
     "media-playback-quality": {
         "href": "https://wicg.github.io/media-playback-quality/",
         "title": "Media Playback Quality Specification",
         "status": "CG-DRAFT",
         "publisher": "WICG"
+    },
+    "secure-contexts": {
```

Comment:
I think I can probably delete this - we get the biblio generated automatically thanks to use of [[!secure-contexts]] right?